spec/integration/worker_metrics_spec.rb

```diff
@@ -130,7 +130,7 @@ def process_jobs(worker, num)
 
       def shutdown_metrics_server
         # Yabeda does not expose this
-        ::Rack::Handler::WEBrick.shutdown
+        ::Rackup::Handler::WEBrick.shutdown
```

Comment:
Rackup was part of Rack, then moved to an independent gem. Rack 3 compatible apps use Rackup.